master/getting-started/openstack/upgrade/setup.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Installing and configuring calico-upgrade
+canonical_url: 'https://docs.projectcalico.org/master/getting-started/openstack/upgrade/setup'
+---
+
+{% include {{page.version}}/install-calico-upgrade.md %}
+
+## Configuring calico-upgrade
+
+### About configuring calico-upgrade
+
+You must configure `calico-upgrade` so that it can connect to both of the
+following:
+
+- [The existing etcdv2 datastore used by {{site.prodname}} v2.6.x](#configuring-calico-upgrade-to-connect-to-the-etcdv2-datastore)
+
+- [The etcdv3 cluster you plan to use for {{site.prodname}} v3.1](#configuring-calico-upgrade-to-connect-to-the-etcdv3-cluster)
+
+{% include {{page.version}}/config-calico-upgrade-etcd.md %}
+
+### Set the version flag
```

Comment:
This section is a little odd here....it is not really about the calico-upgrade tool and sitting here at the bottom it will be easy to miss. What I would suggest with this section is to add it to the top of the page as a section called "Before you begin" That can be a quick fix for this.

Reply:
Also, since it is just a one-item list....it is, by definition, not a list. 😸 So, should be just regular body text, thanks!!